src/lib/input/_input.scss

```diff
@@ -1,6 +1,9 @@
 @import '../core/style/variables';
 @import '../core/style/vendor-prefixes';
 
+
+@mixin mat-input-styles {
```

Comment:
I don't follow why you're turning this into a mixin rather than just moving the file completely

Reply:
My first thought was to move the file but I was curious as to the aesthetics of using a mixin with Bazel since the styles are associated with `mat-input` and thought I would put this up as an option. Probably better to just move the file as originally planned.